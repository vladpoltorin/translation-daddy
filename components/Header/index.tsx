import * as React from 'react';

const Header: React.FunctionComponent = () => <h1>Translation Daddy</h1>;

export default Header;