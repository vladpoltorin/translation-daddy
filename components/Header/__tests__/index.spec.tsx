import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Header from '../';

describe('Header', () => {
  it('Should render header correctly', () => {
    const wrapper = shallow(<Header />);
    expect(toJson(wrapper)).toMatchSnapshot()
  });
});
