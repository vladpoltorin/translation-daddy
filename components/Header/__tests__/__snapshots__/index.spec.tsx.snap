// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Should render header correctly 1`] = `
<h1>
  Translation Daddy
</h1>
`;
